doc/crud.md

```diff
@@ -234,6 +319,11 @@ CDTDocumentRevision *updated = [datastore updateDocumentFromRevision:retrieved
                                                                error:&error];
 
 ```
+```swift
+let retrieved = try datastore.getDocumentWithId("mydoc")
+retrieved.attachments.removeValueForKey("cute_cat.jpg")
```

Comment:
Yes, see the [doc](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Reference/Swift_Dictionary_Structure/index.html#//apple_ref/swift/structm/Dictionary/s:FVSs10Dictionary17removeValueForKeyu0_Rq_Ss8Hashable_FRGS_q_q0__Fq_GSqq0__)